src/components/button/button.ts

```diff
@@ -37,15 +55,15 @@ export class MdButton {
   /** Whether a mousedown has occurred on this element in the last 100ms. */
   isMouseDown: boolean = false;
 
-  setClassList() {return `md-${this.color}`;}
-
   setMousedown() {
     // We only *show* the focus style when focus has come to the button via the keyboard.
     // The Material Design spec is silent on this topic, and without doing this, the
     // button continues to look :active after clicking.
     // @see http://marcysutton.com/button-focus-hell/
     this.isMouseDown = true;
-    TimerWrapper.setTimeout(() => { this.isMouseDown = false; }, 100);
+    TimerWrapper.setTimeout(() => {
```

Comment:
This is because `TimerWrapper.setTimeout()` expects the function to be void. Dart is stricter than TypeScript in that it strictly enforce return values of callbacks. In TypeScript the return value is simply ignored. Please note that there are proposals to enforce stricter rules in TypeScript as well.